test/hotspot/gtest/utilities/test_rbtree.cpp

```diff
@@ -507,6 +541,69 @@ TEST_VM_F(RBTreeTest, NodeStableAddressTest) {
   this->test_stable_nodes_addresses();
 }
 
+TEST_VM_F(RBTreeTest, LeftMostRightMost) {
+  this->test_leftmost_rightmost();
+}
+
+struct PtrCmp {
+  static int cmp(const void* a, const void* b) { return a == b ? 0 : (a > b ? 1 : -1); }
```

Comment:
I'm fairly sure that it's UB to compare pointers with `>`, can you cast these to `uintptr_t` so we don't have to take the ubsan fix review later?